text-ui-test/EXPECTED.TXT

```diff
@@ -1,10 +1,5 @@
 Hello from
```

Comment:
The `help` and `summary` command should probably have text-ui-test before merging.

Reply:
Will try, was getting errors with the fancy fonts (tutorial mode) and table for command summary